core/server/master/src/main/java/alluxio/master/metastore/heap/HeapInodeStore.java

```diff
@@ -122,6 +122,15 @@ public void clear() {
     mEdges.clear();
   }
 
+  /**
+   * Gets all children for the provided edge IDs.
+   * <p>
+   * Returns a map with all children mapped to the provided edge {@code id}.
+   * If no child is found, returns an {@link Collections#emptyMap}.
+   *
+   * @param id the edge ID the children map that will be returned are mapped to
```

Comment:
this description is a little confusing, maybe changing the voice of the sentence.